packages/lime-system/files/usr/lib/lua/lime/network.lua

```diff
@@ -538,4 +580,94 @@ function network.createMacvlanIface(baseIfname, linuxName, argsDev, argsIf)
 	return owrtInterfaceName, linuxName, owrtDeviceName
 end
 
+--! Create a static interface at runtime via ubus
+function network.createStatic(linuxBaseIfname)
```

Comment:
Could you add as a comment in the code the difference between `network.createStatic` and `network.createStaticIface`?

From what I can understand, the difference is that createStaticIface creates the configuration file (the actual interface will appear upon reboot or restart of the networking service), with a custom netmask, gateway and IPv4 or IPv6. While this creates the interface in the running system (not in the configuration file) with fixed netmask, no gateway and the primary IPv4. Right? If this is right, I think that it should have a more informative name than just `createStatic`.